File: OPENAPI-REST-API/openapi-client/cpp-tiny/lib/Models/VpsBackupRows.cpp


#include "VpsBackupRows.h"

using namespace Tiny;

VpsBackupRows::VpsBackupRows()
{
}

VpsBackupRows::VpsBackupRows(std::string jsonString)
{
	this->fromJson(jsonString);
}

VpsBackupRows::~VpsBackupRows()
{

}

void
VpsBackupRows::fromJson(std::string jsonObj)
{
    bourne::json object = bourne::json::parse(jsonObj);


}

bourne::json
VpsBackupRows::toJson()
{
    bourne::json object = bourne::json::object();


    return object;

}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/vps_backup_rows.dart
part of swagger.api;

class VpsBackupRows {
  
  VpsBackupRows();

  @override
  String toString() {
    return 'VpsBackupRows[]';
  }

  VpsBackupRows.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
  }

  Map<String, dynamic> toJson() {
    return {
     };
  }

  static List<VpsBackupRows> listFromJson(List<dynamic> json) {
    return json == null ? new List<VpsBackupRows>() : json.map((value) => new VpsBackupRows.fromJson(value)).toList();
  }

  static Map<String, VpsBackupRows> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, VpsBackupRows>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new VpsBackupRows.fromJson(value));
    }
    return map;
  }
}
